tensorflow/lite/micro/memory_helpers_test.cc

```diff
@@ -31,6 +31,12 @@ void* FakeAllocatePersistentBuffer(TfLiteContext* context, size_t bytes) {
   return reinterpret_cast<void*>(global_persistent_buffer);
 }
 
+// Test structure to test AlignSizeUp.
+struct alignas(32) TestAlignSizeUp {
+  // Opaque blob
+  float blob_data[4];
+};
```

Comment:
let's move this struct inside the test case.

Reply:
Done